oktapam/convert/security_policy_privileges.go

```diff
@@ -118,7 +117,8 @@ func SecurityPolicyRulePrivilegeContainerFromModelToSDK(ctx context.Context, in
 		privilegeType = pam.SecurityPolicyRulePrivilegeType_PASSWORD_CHECKOUT_SSH
 
 	} else {
-		panic("missing stanza in SecurityPolicyRulePrivilegeContainerFromModelToSDK")
+		diags.AddError("missing stanza in OktaPAM provider", "missing stanza in SecurityPolicyRulePrivilegeContainerFromModelToSDK")
```

Comment:
I think schema validation will mostly capture these types of error as it's marked as `Required` in schema. Right?

Reply:
I got it now by going over schema. So, privilege container is required but content inside that can be missing.

Reply:
As this error is user facing, I think we should refine this error. Something like - 

diags.AddError("invalid configuration for privileges[1]", "Privilege is empty. Each privilege must define exactly one type ('Type1', 'Type2'.)")

Reply:
It is user-facing, but it only happens if one of our developers doesn't add an extra stanza here to convert a privilege from terraform back to the sdk. Because it's a programming mistake, I was tempted to leave them as `panic`, but TF has a nice enough error handling system (with diagnostics) that I felt letting TF cleanup properly on an error was probably a better bet.

Reply:
This error will also come if let's say you have config like this below - 

```
      privileges = [
        {
        }, 
        {
          principal_account_ssh = {
            principal_account_ssh   = true
            admin_level_permissions = true
          }
        }
      ]
```